src/mol-plugin/behavior/dynamic/representation.ts

```diff
@@ -186,8 +186,12 @@ export const SelectLoci = PluginBehavior.create({
                         Structure.areEquivalent(structure, oldStructure) &&
                         Structure.areHierarchiesEqual(structure, oldStructure)) return;
 
-                    const reprs = this.ctx.state.data.select(StateSelection.Generators.ofType(SO.Molecule.Structure.Representation3D, ref));
-                    for (const repr of reprs) this.applySelectMark(repr.transform.ref, true);
+                    const children = this.ctx.state.data.select(StateSelection.children(ref));
```

Comment:
@dsehnal  add .childrenOfType helper?

Reply:
``StateSelection.children(ref).ofType(SO.Molecule.Structure.Representation3D)`` should do exactly that.